modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/node/MoveNodeCommand.java

```diff
@@ -274,10 +271,7 @@ private void verifyNoExistingAtNewPath( final NodePath newParentPath, final Node
     {
         final NodePath newNodePath = NodePath.create( newParentPath, newNodeName.toString() ).build();
 
-        CheckNodeExistsCommand.create( this ).
-            nodePath( newNodePath ).
-            throwIfExists().
-            build().
+        CheckNodeExistsCommand.create( this ).nodePath( newNodePath ).throwIfExists().mode( CheckNodeExistsCommand.Mode.SPEED ).build().
```

Comment:
For "top" nodes we better use ACCURACY, for others we can simplify to SPEED, or don't' do a check at all.